src/scenes/VapaaSivistystyo/components/LupaHistoryItem.js

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import {useIntl} from "react-intl";
+import moment from 'moment';
+import styled from "styled-components";
+import Media from "react-media";
+
+import { MEDIA_QUERIES } from "../../../modules/styles";
+
+import { API_BASE_URL } from "../../../modules/constants";
+import { Td, Tr } from "../../../modules/Table";
+import common from "../../../i18n/definitions/common";
+
+const LupaText = styled.span`
```

Comment:
We could use TailwindCSS's atomic css classes for styling instead of pixel values and external style related components like LupaText.